### PR TITLE
Updated Constitution

### PR DESCRIPTION
**Pull request by linamnt**

### CODE REVIEW

File: Group_Constitution [hunk 1]
```diff
@@ -0,0 +1,127 @@
+Constitution of “Mozilla Science Study Group- University of Toronto”
+
+1.	Name
+
+The official name of this recognized campus group is “     ”
```
Comment (lwjohnst86) [inline Group_Constitution:5]: Thanks for getting this going again @linamnt! 

The name we are likely going with is "U of T Scientific Coders" or "University of Toronto Scientific Coders" (see https://github.com/UofTCoders/studyGroup/pull/40 for the discussion on it).

File: Group_Constitution [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+
+The acronym or abbreviation of this group is:
```
Comment (lwjohnst86) [inline Group_Constitution:7]: I don't think we need an abbrev... if we do, maybe SciCoders?

File: Group_Constitution [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+1.	Name
+
+The official name of this recognized campus group is “     ”
+
+The acronym or abbreviation of this group is:      
+
+2.	Purpose and Objectives
+
+The purpose, objectives, mission and/or mandate of organization is outlined here:
+
```
Comment (lwjohnst86) [inline Group_Constitution:12]: I made a comment of this in the previous PR for the constitution:

Some suggestions:

> - To share and learn about other coding techniques and best practices for computing and analysis
> - To improve the collective competence and skills of researchers-in-training at the University of Toronto
> - To build a community of like-minded individuals interested in teaching and learning about coding/programming in the context of scientific research
> - To increase the awareness and need to learn proper programming and coding principals in order to improve the science that uses these techniques

The last one may just be specific to my field (and in general biomedical sciences), but there is a real lack of knowledge about how important programming/coding is for research and science... or even a lack of understanding that programming exists! (partly in jest)

Comment (linamnt) [inline Group_Constitution:12]: Oh sorry, I edited it all in a .doc file so I don't think the changes are
showing! I'll do it in the text file asap.

On Tue, Sep 8, 2015 at 3:08 PM, Luke W Johnston notifications@github.com
wrote:

> In Group_Constitution
> https://github.com/UofTCoders/studyGroup/pull/44#discussion_r38966726:
> 
> > @@ -0,0 +1,127 @@
> > +Constitution of “Mozilla Science Study Group- University of Toronto”
> > +
> > +1. Name
> > +
> > +The official name of this recognized campus group is “     ”
> > +
> > +The acronym or abbreviation of this group is:
> > +
> > +2. Purpose and Objectives
> > +
> > +The purpose, objectives, mission and/or mandate of organization is outlined here:
> > +
> 
> I made a comment of this in the previous PR for the constitution:
> 
> Some suggestions:
> - To share and learn about other coding techniques and best practices
>   for computing and analysis
> - To improve the collective competence and skills of
>   researchers-in-training at the University of Toronto
> - To build a community of like-minded individuals interested in
>   teaching and learning about coding/programming in the context of scientific
>   research
> - To increase the awareness and need to learn proper programming and
>   coding principals in order to improve the science that uses these techniques
> 
> The last one may just be specific to my field (and in general biomedical
> sciences), but there is a real lack of knowledge about how important
> programming/coding is for research and science... or even a lack of
> understanding that programming exists! (partly in jest)
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/UofTCoders/studyGroup/pull/44/files#r38966726.

## 

Lina Tran
MSc. Candidate | Frankland Lab
Department of Physiology
University of Toronto

File: Group_Constitution [hunk 4]
```diff
@@ -0,0 +1,127 @@
...
+•	Handle official correspondence of the organization
+Additional responsibilities may include: 
+     
+
+
+
+The Treasurer shall:
+•	Record all financial transactions of the group
+•	Hold signing officer authority along with the President for financial purposes
+•	Maintain a budget of income and expenses along with receipts
```
Comment (lwjohnst86) [inline Group_Constitution:63]: > - Maintain a _balanced_ budget of income and expenses
> - Keep a record of receipts in both hard-copy _and_ electronic (scanned)

File: Group_Constitution [hunk 5]
```diff
@@ -0,0 +1,127 @@
...
+•	Advise members on financial position of the group
+•	Prepare an annual budget for the group as well as budgets for specific events
```
Comment (lwjohnst86) [inline Group_Constitution:65]: I would also add:

> - Reimburse members of expense claims in a timely fashion.

**Comment by lwjohnst86** [comment]
Thanks so much @linamnt ! Just for next time if you want to do two or more pull requests and the previous one hasn't been merged yet, follow the advice in [this wiki](https://github.com/UofTCoders/studyGroup/wiki/Contributing-a-lesson) to create two branches for two separate pull requests. 

For now, I think it would be best to split the pull request into two separate ones. So can you follow these commands? @QuLogic can you confirm? I have tested these out on your repo @linamnt, so it should work.

```
git branch logos 156ad7
git checkout logos
git cherry-pick af09e9e
git cherry-pick 94e9d97
git cherry-pick 13d5b27
git pull upstream gh-pages ## Make sure you have an upstream remote (git remote -v)
git push origin logos
git checkout gh-pages ## to get back to main branch
```

You can now make a new pull request in GitHub of the `logos` branch. This won't remove the commits from this pull request, but we can merge the `logos` PR sooner than the constitution PR, which you can then update.

PS: The dark icons look awesome!

**Comment by linamnt** [comment]
Okay, I think it worked! Thanks, git is so full of wonderful tricks :D

On Thu, Sep 17, 2015 at 11:10 AM, Luke W Johnston notifications@github.com
wrote:

> Thanks so much @linamnt https://github.com/linamnt ! Just for next time
> if you want to do two or more pull requests and the previous one hasn't
> been merged yet, follow the advice in this wiki
> https://github.com/UofTCoders/studyGroup/wiki/Contributing-a-lesson to
> create two branches for two separate pull requests.
> 
> For now, I think it would be best to split the pull request into two
> separate ones. So can you follow these commands? @QuLogic
> https://github.com/QuLogic can you confirm? I have tested these out on
> your repo @linamnt https://github.com/linamnt, so it should work.
> 
> git branch logos 156ad7
> git checkout logos
> git cherry-pick af09e9e
> git cherry-pick 94e9d97
> git cherry-pick 13d5b27
> git pull upstream gh-pages ## Make sure you have an upstream remote (git remote -v)
> git push origin logos
> git checkout gh-pages ## to get back to main branch
> 
> You can now make a new pull request in GitHub of the logos branch. This
> won't remove the commits from this pull request, but we can merge the
> logos PR sooner than the constitution PR, which you can then update.
> 
> PS: The dark icons look awesome!
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/UofTCoders/studyGroup/pull/44#issuecomment-141117070.

## 

Lina Tran
MSc. Candidate | Frankland Lab
Department of Physiology
University of Toronto

**Comment by lwjohnst86** [comment]
@QuLogic @mbonsma @linamnt Hey is this good to merge in yet? We got official status right?

**Comment by mbonsma** [comment]
Yeah I think we can merge. We're now a recognized group.

**Pull request merged**